.github/workflows/lint.yml

```diff
@@ -14,10 +14,6 @@ on:
 permissions:  # set top-level default permissions as security best practice
   contents: read
 
-concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name == 'workflow_dispatch' }}
-  cancel-in-progress: true
-
```

Comment:
We can keep this so PRs updates do not trigger an excessive amount of lint jobs

Reply:
Good point. Lint CI is a fast one and it shouldn't be skipped frequently in the main branch.